client/libs/ui-libraries/src/lib/ui-table/ui-table.component.scss

```diff
@@ -34,3 +33,13 @@
     border: none;
   }
 }
+
+.hide-scroll {
```

Comment:
We could probably put this in `hide-scroll` in the `_utilities.scss` file.
Also you can refactor this like so:
```scss
.hide-scroll {
  &::-webkit-scrollbar {
    display: none;
  }
  overflow-x: scroll;
  -ms-overflow-style: none;
}
```